jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -74,3 +105,153 @@ def vue_save_figure(self, filetype):
         the bqplot.Figure save methods.
         """
         self.save_figure(filetype=filetype)
+
+    def _save_movie(self, i_start, i_end, fps, filename, rm_temp_files):
+        # NOTE: All the stuff here has to be in the same thread but
+        #       separate from main app thread to work.
+        self.app.loading = True  # Grays out the app
+        viewer = self.viewer.selected_obj
+        slice_plg = self.app._jdaviz_helper.plugins["Slice"]._obj
+        orig_slice = slice_plg.slice
+        ts = float(slice_plg.play_interval) * 1e-3  # ms to s
+        temp_png_files = []
+
+        # TODO: Expose to users?
+        i_step = 1  # Need n_frames check if we allow tweaking
+
+        # No wrapping. Forward only.
+        if i_start < 0:
+            i_start = 0
+        if i_end > slice_plg.max_value:
+            i_end = slice_plg.max_value
+        if i_end <= i_start:
+            raise ValueError(f"No frames to write: i_start={i_start}, i_end={i_end}")
+
+        i = i_start
+        while i <= i_end:
+            slice_plg._on_slider_updated({'new': i})
+            cur_pngfile = f"._cubeviz_movie_frame_{i}.png"
+            self.save_figure(filename=cur_pngfile, filetype="png")
+            temp_png_files.append(cur_pngfile)
+            i += i_step
+            time.sleep(ts)  # Avoid giving user epilepsy

```

Comment:
pulling this from the slice plugin feels weird to me and could result in movies taking forever to export with no clear reason to the user.  Is no sleep really too fast?  Is epilepsy really a concern for realistic data, a non-fullscreen-viewer, and a greyed out application?

Reply:
This answered the first question I had when doing my first test run - I'm trying to save out the whole movie, and it's taking forever (ok, not forever, but 5-10 minutes) to step through the cube, and I wasn't sure if that was really the time it's taking to write the movie or not. I guess it's due to this sleep and maybe writing the movie is much faster? Maybe it's ok to have a sleep here, but I think it could be 5-10x faster than it is now.

Reply:
My preference would be to remove the sleep for now and re-introduce it if it ever becomes a problem (although I'd prefer to just further gray-out and save the time).  The other option might be to pull the sleep from the FPS setting in this plugin rather than the slice plugin?

Reply:
Okay I can disable that. But the second sleep to wait for bqplot has to stay.

Reply:
Yes, but maybe something hardcoded and very small?

Reply:
Mario originally proposed 0.1 sec per loop, is that small enough?

Reply:
I deleted this sleep and changed the next one to 0.05 and it seemed more reasonable. Definitely not too fast.

Reply:
Honestly even just sitting in a while-pass loop without a sleep would probably be fine, but anything not-noticeable (sub 0.1 sec) should work too.

Reply:
Extra sleep is removed. The remaining sleep is now 0.05 as Ricky suggested.